astropy/coordinates/tests/test_velocity_corrs.py

```diff
@@ -393,3 +396,36 @@ def _get_barycorr_bvcs_withvels(coos, loc, injupyter=False):
                            elevation=loc.geodetic[2].to(u.m).value)
         bvcs.append(res)
     return bvcs*u.m/u.s
+
+
+@pytest.mark.remote_data
+def test_regression_XXXX():
+    """
+    Make sure that when we include the proper motion and radial velocity of
+    a SkyCoord, our velocity corrections remain close to TEMPO2.
+
+    We check that tau Ceti is within 5mm/s
+    """
+    # Wright & Eastman (2014) Table2
+    # Corrections for tau Ceti
+    reduced_jds, tempo2, barycorr = np.loadtxt(
+        download_file('http://astroutils.astronomy.ohio-state.edu/exofast/pro/exofast/bary/zb.txt')).T
```

Comment:
the PR to astropy-data has been merged, so please use that copy instead

Reply:
Thanks @bsipocz ! Done.